cmd/buildah/bud.go

```diff
@@ -179,6 +183,7 @@ func budCmd(c *cli.Context) error {
 		Compression:         imagebuildah.Gzip,
 		Quiet:               c.Bool("quiet"),
 		SignaturePolicyPath: c.String("signature-policy"),
+		SkipTLSVerify:       c.Bool("tls-verify"),
```

Comment:
Since tls-verify is defaulted to true, shouldn't this be 
SkipTLSVerify:  ! c.Bool("tls-verify")

Reply:
Good point.  I just grabbed the name suggestion from Nalin, plugged it in, and didn't consider that.  Will adjust per your suggestions.

Reply:
Fixed